Matterhook.NET.MatterhookClient/MattermostAttachment.cs

```diff
@@ -144,31 +112,27 @@ public class MattermostAttachment
         [JsonProperty(PropertyName = "thumb_url")]
         public string ThumbUrl { get; set; }
 
-
         #endregion
-    }
 
-    /// <summary>
-    /// https://docs.mattermost.com/developer/message-attachments.html#fieldshttps://docs.mattermost.com/developer/message-attachments.html#fields
-    /// </summary>
-    public class MattermostField
-    {
-        /// <summary>
-        /// A title shown in the table above the value.
-        /// </summary>
-        [JsonProperty(PropertyName = "title")]
-        public string Title { get; set; }
-
-        /// <summary>
-        /// The text value of the field. It can be formatted using Markdown.
-        /// </summary>
-        [JsonProperty(PropertyName = "value")]
-        public string Value { get; set; }
-
-        /// <summary>
-        /// Optionally set to “True” or “False” to indicate whether the value is short enough to be displayed beside other values.
-        /// </summary>
-        [JsonProperty(PropertyName = "short")]
-        public bool Short { get; set; }
+        public MattermostAttachment Clone()
+        {
+                var outAtt = new MattermostAttachment
+                {
+                    AuthorIcon = this.AuthorIcon,
+                    AuthorLink = this.AuthorLink,
+                    AuthorName = this.AuthorName,
+                    Color = this.Color,
+                    Fallback = this.Fallback,
+                    Fields = this.Fields,
+                    ImageUrl = this.ImageUrl,
+                    Pretext = this.Pretext,
+                    ThumbUrl = this.ThumbUrl,
+                    Title = this.Title,
+                    TitleLink = this.TitleLink,
+                    Actions = this.Actions,
+                    Text = this.Text
```

Comment:
As per original `CloneAttachment()` method, `Text` should be blank 
```suggestion
                    Text = ""
```

Reply:
I didnt let Text blank, since when cloning an object, I expect to get an exact copy of the object.

Reply:
The reason for the text to be blank on the "cloned" message, is to do with the message splitting. The `Text` property of the new cloned message should not be the same as the original message, rather it will be whatever the text splitting function decides it to be... if that makes sense?

Reply:
yes I understand, but in the code, when splitting, the message gets cloned first (with same Text as original) and then you set the Text property to the proper chunck of text (overwriting the original). 

I understand the sense of the clone function, but when a user meets a method named `clone`, I think it expects and exact copy of the object. So maybe a solution would to implement a `Clone` and a `CloneWithoutText` / `CloneMetadataOnly`  or just a `Clone(bool onlyMetadata)` where it is clear what the function does.